src/suit_directive.c

```diff
@@ -405,6 +417,14 @@ int suit_directive_process_dependency(struct suit_processor_state *state, struct
 			retval = suit_processor_load_envelope(state, envelope_str, envelope_len);
 		}
 
+		if (retval == SUIT_SUCCESS) {
+			SUIT_DBG("Authorize execution of sequence\r\n");
+			retval = suit_plat_authorize_process_dependency(
```

Comment:
Just a few notes:

- The API has the directive name in it (process-dependency), so adding it in other places will require a better name.
- On one hand, it might be good to fail early during integrity validation if the manifest cannot be processed. On the other hand, it would limit the usability of the dependency-integrity condition.
- One use case that is rather unusual is the fact that with the dependency-integrity we may check if the system has a certain manifest installed from the manifest sequence. If we add the platform call there, such checks will need to follow the restricted hierarchy (i.e. radio manifest cannot ask about application manifest).